assets/css/root.scss

```diff
@@ -31,12 +31,12 @@
 
     --bs-nav-link-color: var(--text-color) !important;
     --bs-navbar-color: var(--text-color);
+  }
 
-    .modal {
-      --bs-modal-bg: var(--secondary-background);
-      --bs-border-color: var(--secondary-background);
-      --modal-color: var(--text-color);
-    }
+  dialog {
+    --bs-modal-bg: var(--secondary-background);
+    --bs-border-color: var(--secondary-background);
+    --modal-color: var(--text-color);
   }
```

Comment:
Maybe we can add the fade animation like this?

```css
  dialog[open] {
    animation: fadein .15s ease-in forwards;
  }
  
  @keyframes fadein {
    0% {
      opacity: 0;
    }
    
    100% {
      opacity: 1;
    }
  }
```

Reply:
Added it to page.scss